frontend/lib/src/util/utils.ts

```diff
@@ -563,7 +563,7 @@ export function extractPageNameFromPathName(
   // regex special-characters. This is why we're stuck with the
   // weird-looking triple `replace()`.
   return decodeURIComponent(
-    document.location.pathname
+    pathname
```

Comment:
Using `document.location.pathname` here seems to have been an accident as we pass `document.location.pathname` to this helper function where it's used as the `pathname` variable but then proceed to never use the variable.

Reply:
oh :( I think we might want to add a linting rule to catch unused parameters. I added it as tech debt [here](https://www.notion.so/snowflake-corp/Evaluate-activating-additional-frontend-linting-rules-acda1a63931a4cafbd7c3df6a5b54d32) (cc @sfc-gh-bnisco)